mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraMessageModule.java

```diff
@@ -56,6 +56,7 @@ public CassandraMessageModule() {
                     .addClusteringColumn(CassandraMessageIds.MAILBOX_ID, timeuuid())
                     .addClusteringColumn(CassandraMessageIds.IMAP_UID, bigint())
                     .addColumn(MessageIdToImapUid.MOD_SEQ, bigint())
+                    .addColumn(MessageIdToImapUid.MOD_SEQ_SET, set(bigint()))
```

Comment:
Read old+new, write new. 

We won't had problems. Just need to alter the tables before James reboot.